### ISSUE TITLE
Cannot find module '../lib/unicode'

### ISSUE THREAD

**Issue by ShiiRochi** [webpack/loader-utils#160]
Today I have to work on my PC (running Window) and I cannot start react app because of an error printed below.

It says that it cannot find '../lib/unicode', error move to that package.

```
yarn run v1.9.4
$ node scripts/start.js
internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module '../lib/unicode'
Require stack:
- D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\node_modules\json5\lib\util.js
- D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\node_modules\json5\lib\parse.js
- D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\node_modules\json5\lib\index.js
- D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\node_modules\loader-utils\lib\parseQuery.js
- D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\node_modules\loader-utils\lib\getOptions.js
- D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\node_modules\loader-utils\lib\index.js
- D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\index.js
- D:\GITHUB_REPOS\vesna-soft\univey\config\webpack.config.js
- D:\GITHUB_REPOS\vesna-soft\univey\scripts\start.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (D:\GITHUB_REPOS\vesna-soft\univey\node_modules\html-webpack-plugin\node_modules\json5\lib\util.js:1:258)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Module.require (internal/modules/cjs/loader.js:852:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\node_modules\\html-webpack-plugin\\node_modules\\json5\\lib\\util.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\node_modules\\html-webpack-plugin\\node_modules\\json5\\lib\\parse.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\node_modules\\html-webpack-plugin\\node_modules\\json5\\lib\\index.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\node_modules\\html-webpack-plugin\\node_modules\\loader-utils\\lib\\parseQuery.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\node_modules\\html-webpack-plugin\\node_modules\\loader-utils\\lib\\getOptions.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\node_modules\\html-webpack-plugin\\node_modules\\loader-utils\\lib\\index.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\node_modules\\html-webpack-plugin\\index.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\config\\webpack.config.js',
    'D:\\GITHUB_REPOS\\vesna-soft\\univey\\scripts\\start.js'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

This line `var _unicode=require('../lib/unicode')` from utils throws error.

**Comment by evilebottnawi** [comment]
Please create reproducible test repo

**Comment by ShiiRochi** [comment]
This is a commercial project, I cannot publish it)) Also it will take time to create repo, where that problem could be reproducible.

**Comment by evilebottnawi** [comment]
I can't help without reproducible steps or repo, just think how I can help with this if I can’t understand how to get an error :smile:

**Comment by ShiiRochi** [comment]
I understand that. Just for now I have information that it tries to require('../lib/unicode'), however either in that package, nor in html-webpack-plugin, that can be also seen in logs, anything that could be called like 'unicode' does not exist in any lib folder. Maybe it is Windows' super feature.

**Comment by evilebottnawi** [comment]
Maybe you accidentally remove something from node_modules, can you reinstall?

**Comment by ShiiRochi** [comment]
> Maybe you accidentally remove something from node_modules, can you reinstall?

No, I just pulled locally all changes from repo and ran `yarn install`. After three cycles of remove node_modules and new install, nothing changed. I made a fresh clone. The same thing.

**Comment by evilebottnawi** [comment]
Can you provide yarn version?

**Comment by ShiiRochi** [comment]
Sorry, I watched an error again. It's a problem with `json5` package inside html-webpack-plugin. I tried to downgrade it to `3.2.0` and got the same error, but in react-dev-utils with `json5` also. And really, in json5 in lib folder I have unicode.js, but in node_modules inside html-webpack-plugin I do not see it.

**Comment by evilebottnawi** [comment]
Please create reproducible test repo, maybe bug in `html-webpack-plugin`

**Comment by ShiiRochi** [comment]
I went through the problem by creating fresh ejected react-app. I combined fresh package.json with my current one. Removed `node_modules` in repo folder and moved `node_modules` from fresh ejected app folder to repo folder. 

After running `yarn install` and scripts update in `scripts` folder according to fresh ejected app scripts. I got a working app.

I still don't understand what was that...so, maybe some enthusiasts will find out that, but if someone for some reason would have the same thing happening, it can be resolved in the described above way.
